src/homepage/Hero.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
```

Comment:
Looks like `useEffect` isn't being used in this file. Can we remove it?

Reply:
Good catch. Removed.